### PR TITLE
[Privly.js] Removed code duplication

### PR DESCRIPTION
**Pull request by hitesh96db**
Similar to https://github.com/privly/privly-chrome/pull/107

### CODE REVIEW

File: chrome/content/privly.js [hunk 1]
```diff
@@ -623,27 +623,28 @@ var privly = {
...
+    "use strict";
+
+    this.toggleInjectionHelper(document.getElementsByTagName("iframe"), "");
+    this.toggleInjectionHelper(document.getElementsByTagName("a"), "inherit");
+  },
+
+  /*
+   * Helper function for toggling the display of links and iframes
+   */
+  toggleInjectionHelper: function(elements, displayStyle) {
```
Comment (smcgregor) [inline chrome/content/privly.js]: Needs function header comments for the parameters.

**Comment by hitesh96db** [comment]
@smcgregor Added "param" to both Chrome and Firefox PR.

**Comment by smcgregor** [comment]
You haven't pushed something yet? I am not seeing a change.

File: chrome/content/privly.js [hunk 2]
```diff
@@ -623,27 +623,32 @@ var privly = {
...
+    "use strict";
+
+    this.toggleInjectionHelper(document.getElementsByTagName("iframe"), "");
+    this.toggleInjectionHelper(document.getElementsByTagName("a"), "inherit");
+  },
+
+  /*
+   * Helper function for toggling the display of links and iframes
+   *
+   * @param {array} elements Array of HTML elements to be toggled
```
Comment (hitesh96db) [inline chrome/content/privly.js]: @smcgregor Here

Comment (smcgregor) [inline chrome/content/privly.js]: Your last commit was 12 days ago. You edited history?

Comment (hitesh96db) [inline chrome/content/privly.js]: Nope, I just re-based it with the old commit.

**Pull request merged**